core/api/urls.py

```
@@ -1,9 +1,18 @@
-
 from django.urls import path
-from .views import ProfileView, UserInfoView
+from .views import (
+    ProfileView, UserInfoView, UserContributionView,
+    ContributionAdsView, EnrollmentView, ContributionDetailsView
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)